esp/services/ws_fs/ws_fsBinding.cpp

```diff
@@ -393,12 +393,15 @@ int CFileSpraySoapBindingEx::downloadFile(IEspContext &context, CHttpRequest* re
         if (!context.validateFeatureAccess(FILE_SPRAY_URL, SecAccess_Full, false))
             throw MakeStringException(ECLWATCH_FILE_SPRAY_ACCESS_DENIED, "Failed to download file. Permission denied.");
 
-        StringBuffer netAddressStr, osStr, pathStr, nameStr;
+        StringBuffer netAddressStr, osStr, pathStr, nameStr, dropZoneName;
         request->getParameter("NetAddress", netAddressStr);
         request->getParameter("OS", osStr);
         request->getParameter("Path", pathStr);
         request->getParameter("Name", nameStr);
+        request->getParameter("DropZoneName", dropZoneName);
 
+        if (getDropZoneScopePermissions(context, dropZoneName, pathStr, netAddressStr) < SecAccess_Read)
```

Comment:
To help Ops diagnose problems without our involvement, would it help to include the user name, and the actual LDAP  permission setting?